topo_processor/cli/geostore/export.py

```diff
@@ -0,0 +1,88 @@
+import shutil
+
+import boto3
+import click
+from linz_logger import LogLevel, get_log, set_level
+
+from topo_processor.geostore.invoke import invoke_import_status, invoke_lambda
+from topo_processor.util.configuration import temp_folder
+from topo_processor.util.time import time_in_ms
+
+
+@click.command()
+@click.option(
+    "-s",
+    "--source",
+    required=True,
+    help="The s3 path to the collection.json of the survey to export",
+)
+@click.option(
+    "-sid",
```

Comment:
how does this work? How does this handle `-s -i -d` ? 

generally these should be a single letter?